jsk_arc2017_baxter/euslisp/lib/apc-interface.l

```diff
@@ -0,0 +1,376 @@
+;; -*- mode: lisp;-*-
+
+(require "package://jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l")
+(require "package://jsk_2015_05_baxter_apc/euslisp/jsk_2015_05_baxter_apc/util.l")
+(require "package://jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter.l")
+(require "package://jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/util.l")
+(require "package://jsk_arc2017_baxter/euslisp/lib/util.l")
+
+(ros::load-ros-manifest "jsk_arc2017_baxter")
+
+(unless (find-package "JSK_ARC2017_BAXTER")
+  (make-package "JSK_ARC2017_BAXTER"))
+
+(defclass jsk_arc2017_baxter::transform-listener
+  :super ros::transform-listener)
+
+(defmethod jsk_arc2017_baxter::transform-listener
+  (:tf-pose->coords
+    (frame_id pose)
+    (send (send self :lookup-transform "base" frame_id (ros::time 0))
+          :transform (ros::tf-pose->coords pose))))
+
+(defclass jsk_arc2017_baxter::apc-interface
+  :super jsk_2016_01_baxter_apc::baxter-interface
```

Comment:
Even if you set `*ri*` to the slots, you can use methods from `baxter-interface` in jsk_apc2016, right?
What is the problem?